kythe-verification/test.sh

```diff
@@ -55,7 +55,7 @@ for fut in \
     "$BASIC/TypeDef.hs"
 do
   echo "Verifying: $fut"
-  $GHC_KYTHE -- --check_for_singletons=true $fut 2> /dev/null | $VERIFIER -goal_prefix '-- -' $fut \
```

Comment:
Was this a bug?

Reply:
Yes, was passing the flag to GHC instead of the verifier. Now it makes the verifier fail if there's a unification variable that is only defined but otherwise not used (or maybe plainly just mentioned once?).